commonui/src/main/res/values/strings.xml

```diff
@@ -31,4 +30,9 @@
     <string name="not_translated_in_braces">(not translated)</string>
 
     <string name="authentication_error_header">Authentication Error</string>
+
+    <string name="mobileTransferMethodsHeader">Transfer methods</string>
```

Comment:
Ideally this should not be commonUI module but inside Transfer Method module

Reply:
yeah but it's used in transfers ui and transfer method account ui modules